packages/common/src/styles/slick-bootstrap.scss

```diff
@@ -107,6 +107,27 @@
           font-size: var(--slick-group-totals-formatter-font-size, $slick-group-totals-formatter-font-size);
         }
       }
+
+      &.slick-rbe-editmode.active .slick-cell,
+      &.slick-rbe-editmode .slick-cell {
+        background-color: var(--slick-row-based-edit-editmode-bgcolor, $slick-row-based-edit-editmode-bgcolor);
+
+        &:hover {
+          background-color: var(--slick-row-based-edit-editmode-hover-bgcolor, $slick-row-based-edit-editmode-hover-bgcolor);
+
+          & .active {
```

Comment:
I don't think the space is necessary, right? `&.active` if it is necessary then remove the `&` I think